ESP8266/ESP8266.cpp

```diff
@@ -425,6 +456,56 @@ void ESP8266::_connect_error_handler()
     }
 }
 
+void ESP8266::_no_more_data(const int id)
+{
+    static const uint32_t amount = 0;
+
+    struct packet* packet = (struct packet*) (malloc(
+            sizeof(struct packet) + amount));
+
+    if (!packet) {
+        return;
+    }
+
+    packet->id = id;
+    packet->len = amount;
+    packet->next = 0;
+
+    // append to packet list
+    *_packets_end = packet;
+    _packets_end = &packet->next;
+}
+
+void ESP8266::_connect_socket0_handler()
+{
+    static const int id = 0;
+    _no_more_data(id);
```

Comment:
Is this too verbal? Why not just `_no_more_data(0);`

Reply:
True